bait-files/src/main/java/com/google/sps/servlets/UserHistoryServlet.java

```diff
@@ -24,32 +26,47 @@
 
 @WebServlet("/userHistory")
 public class UserHistoryServlet extends HttpServlet {
+  class SortByMostRecentDate implements Comparator<Review> {
```

Comment:
I don't think you need this.. see suggestion below

Reply:
Changed to default comparator